pkg/cloud/azure/actuators/machine/reconciler.go

```diff
@@ -374,9 +372,12 @@ func (s *Reconciler) Exists(ctx context.Context) (bool, error) {
 	case v1beta1.VMStateUpdating:
 		klog.Infof("Machine %v is updating", to.String(vm.VMID))
 	default:
+		klog.Infof("not found vm for machine %s", s.scope.Machine.GetName())
```

Comment:
we klog with capital letter by convention or prepend the machine Name.

Reply:
fixed